src/main/resources/component-070-biz.conf

```diff
@@ -988,6 +988,9 @@ storage {
         # Note that this check has to be enabled manually using URLBuilder.enableLargeFileDetection.
         largeFileLimit = 128M
 
+        # Number of days to cache virtual URLs. If set to -1, there will be no special handling for virtual URLs.
+        maxCachedDaysForVirtualURL = -1
```

Comment:
**Suggestion**: it would give more flexibility if this setting could be defined per space.